src/components/Navbar.js

```diff
@@ -1,18 +1,19 @@
 import React from 'react';
 import { Link } from 'react-router-dom';
+import '../App.css';
 
 function Navbar() {
   return (
     <div>
       <nav>
-        <h1 className="topic">TechMoves Books  CRM</h1>
-        <ul>
-          <li>
+        <h1 className="topic">BookstoreCRM</h1>
+        <ul className="nav-link">
+          <li className="link-home">
             <Link className="link" to="/">
               Books
             </Link>
           </li>
-          <li>
+          <li className="link-category">
             <Link className="link" to="/categories">
               Categories
             </Link>
```

Comment:
- [ ] Good job styling the navbar but it does not meet the complete design requirements. Please kindly ensure it matches the original design guidelines(spacing, font, color etc).

### Your Design
![image](https://github.com/techmoves/bookstores/assets/43995830/ac341632-7174-4bde-97bb-533153b969d6)

### Original Design
![image](https://github.com/techmoves/bookstores/assets/43995830/9709d2d5-9092-4616-8620-57cf0c273fc6)